routes/index.js

```diff
@@ -70,6 +73,11 @@ module.exports = function(io) {
       voters: [],
     }));
 
+    input.settings = {
+      addChoices: input.addChoices !== 'on',
```

Comment:
shouldn't this be `input.addChoices === 'on'`, since `addChoices` should be `true` if the checkbox was clicked?